R/initialization.R

```diff
@@ -4,82 +4,84 @@
 #' @param stan_data a list of data elements that will be passed to stan
 #' @param params a dataframe of parameter values that are passed to stan
 #' to specify the priors in the model
+#' @param stdev a numeric value indicating the standard deviation to sample
+#' from when initializing, particularly from a standard normal. Also acts as
+#' a multiplier on the prior standard deviation, to restrict the initial value
+#' to be sampled from the center of the prior. Default is 0.01
 #'
 #' @return a list of initial values for each of the parameters in the
 #' `wwinference` model
-get_inits <- function(stan_data, params) {
-  # Assign parmeter names
-  par_names <- colnames(params)
-  for (i in seq_along(par_names)) {
-    assign(par_names[i], as.double(params[i]))
-  }
-
+get_inits <- function(stan_data, params, stdev = 0.01) {
```

Comment:
As written this will only generate one chain worth of inits. Is the intended use something like  

```R
lapply(1:n_chains, function(x){get_inits(stan_data, params)})
```
?

If we're going to have a user-exported function why not have that functionality under the hood, take the number of chains as an argument, and then call a `get_inits_for_one_chain` helper function under the hood.

Doesn't have to be for this PR.

Reply:
Yup, that is what happens I do a for loop in the `wwinference()` function. Can rename.